flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/CreatingExecutionGraphTest.java

```diff
@@ -227,6 +237,16 @@ public void goToWaitingForResources() {
             hadStateTransitionHappened = true;
         }
 
+        @Override
+        public Executor getIOExecutor() {
+            return MoreExecutors.directExecutor();
```

Comment:
```suggestion
            return Executors.directExecutor();
```
from org.apache.flink.util.concurrent.Executors